### ISSUE TITLE
Error on launch and when downloading from Nexus using the mod manager button

### ISSUE THREAD

**Issue by DJWolf1** [LePresidente/modorganizer#240]
# The problem

Every time I launch mod manager and each time I use the "Mod Manager Download" button at Nexus (for Skyrim) I get the following pop-up error:
"The ordinal 4430 could not be located in the dynamic link library LIBEAY32.dll"
This has been happening since I first installed MO 4 days ago.  I don't know if further problems will arise from this nor do I know if I am missing anything since I am in the process of learning. 

# Environment

Mod Organizer 1.3.11  Revision: d1c81666578e+
This is my first introduction to MO
Windows 7 Pro 64bit

# Details

Those are the only details I have.  What problems it is causing I don't know.

# Link to Mod Organizer logs

I would love to give you more information but I don't know what USVFS is or where it is.  I have 5 logs that are called "ModOrganizer xx_xx_xx_xx_xx.log"  Is that them?  Looking at them, they concern my Modding environment and the 70 or so mods I have installed but this error occurred before I had assigned this instance of MO to Skyrim, before I had installed a single mod.  It occurred on launching MO after I had installed it.

I installed it in ...SteamApps/Common/Skyrim.

## MO Interface

I only have one interface log and given that it is four lines it won't be that long:

20:14:23 [D] ssl support: 0
20:14:23 [D] Working directory: E:\Games\SteamApps\common\Skyrim\Mod Organizer
20:14:23 [D] MO at: E:\Games\SteamApps\common\Skyrim\Mod Organizer
20:14:23 [D] not primary instance, sending download message

I apologize that I am not of greater help.  Please, if you need more, tell me what to do and I will do it.

**Comment by SuperSandro2000** [comment]
1st don't install into the game folder.
2nd Only MO2 is in active development. MO1 is as it is.

**Comment by Al12rs** [comment]
This is Mod Organizer 2, not MO1. They are two separate projects. It looks like you either did not install Mo1 correctly or you are missing the dependencies like vcredist that are needed.

**Comment by Al12rs** [comment]
MO1 has no problems with the game folder by the way only MO2.

**Comment by Al12rs** [comment]
Closing this since it refers to different program.

**Comment by LePresidente** [comment]
@DJWolf1 
Mod Organizer is not using the correct ssl dll's, "20:14:23 [D] ssl support: 0" I'll have a look if I can find the correct version and upload here in a bit

**Comment by DJWolf1** [comment]
I realize that you guys are the developer team and that you are focused upon MO for SSE but I downloaded MO1 from Nexus.   Where should I go for MO help?  How do I get "the correct ssl dll's, "20:14:23 [D] ssl support: 0""?  Is this due to some Windows failure of mine?

I just spent 3 months playing SSE and the game was a nightmare.  It looked bad and the changes it implemented were more like controls and restrictions.  It is not the future and it will not survive.  Bethesda claims it supports modding but this is a company that couldn't spend five minutes to clean it's ESMs before release - that was deliberate.  Back in Old Skyrim with the same mods loaded I'm overwhelmed by the desire to continue taking screenshots...  Just a thought.
